templates/minio.service.j2

```diff
@@ -33,5 +39,9 @@ SendSIGKILL=no
 
 SuccessExitStatus=0
 
+{% if minio_server_addr.split(':')[1] < 1024 %}
```

Comment:
This will break with v6 :grin:  `[::1]:80`.

Reply:
should be fixed now :smile: